README.md

```diff
@@ -64,21 +71,41 @@ zip -r -q compiled_all_sql.zip target/compiled_all_sql
 
 ### Integration with Report Portal
 
-https://reportportal.io/ helps you to manage your test launches. Here at EPAM we're using this tool to manage over 4,000 DBT tests  
+https://reportportal.io/ helps you to manage your test launches. Here at EPAM we're using this tool to manage over 4,000 DBT tests
 
 In order to upload your test run to reportportal you can use the following script:
 ```shell
 dbt-junitxml parse target/run_results.json target/manifest.json dbt_test_report.xml
 zip dbt_test_report.zip dbt_test_report.xml
 REPORT_PORTAL_TOKEN=`Your token for Report Portal`
-RESPONSE=`curl -X POST "https://reportportal.io/api/v1/epm-plxd/launch/import" -H  "accept: */*" -H  "Content-Type: multipart/form-data" -H  "Authorization: bearer ${REPORT_PORTAL_TOKEN}" -F "file=@dbt_test_report.zip;type=application/x-zip-compressed"`
+RESPONSE=`curl -X POST "https://reportportal.epam.com/api/v1/plugin/{project_name}/JUnit/import" -H  "accept: */*" -H  "Content-Type: multipart/form-data" -H  "Authorization: bearer ${REPORT_PORTAL_TOKEN}" -F "file=@dbt_test_report.zip;type=application/x-zip-compressed"`
```

Comment:
please keep it `.io` since it's externally facing URL / deployment of Report Portal

Reply:
fixed